pkg/action/loader.go

```diff
@@ -245,6 +247,15 @@ func ConvertInputToTplVars(input Input, ac *DefAction) map[string]interface{} {
 		}
 	}
 
+	// Push predefines variables into values list
+	// @todo add warning when user tries to create arg/opt with the same name.
+	cuser := getCurrentUser()
+	if cuser != "" {
```

Comment:
Please, move out of the function.
Please, implement a default value 0 for user and group. That's needed when running in environments like Windows for the moment.

Reply:
updated, thank you